### DISCUSSION TITLE
mycroft wait_while_speaking() not defined

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#228] [Q&A] [answered]
where do i find the OVOS API list that shows me the replacement for wait_while_speaking()

**Comment by JarbasAl** [comment, marked as answer]
there isnt a standalone `wait_while_speaking()` method anymore

when using `self.speak` or `self.speak_dialog`  you can pass `wait=True` to the method if you need to block until your own speech ends

skills can only wait for their own speech to end
